.github/workflows/docusaurus.yml

```diff
@@ -47,11 +47,17 @@ jobs:
       - name: Build Website
         run: |
           ./build-site.sh
+      - name: Search for Broken Links
+        run: |
+          node ./test-broken-links.js
       - name: Publish Website
         run: |
           # Extract GitHub org/user
           REPO="${{ github.repository }}"
           GIT_USERNAME=${REPO%/*}
+          if [ "$GIT_USERNAME" != "finos" ]
```

Comment:
don't need this. you can use if directly as part of the CI job: see https://docs.github.com/en/actions/using-jobs/using-conditions-to-control-job-execution

we can do something like
https://github.com/finos/legend-studio/blob/master/.github/workflows/release.yml#L17